docs/sources/mimir/_index.md

```diff
@@ -15,7 +15,7 @@ keywords:
   - observability
 ---
 
-# Grafana Mimir documentation
+# Grafana Mimir documentation test
```

Comment:
I assume this is intentional and will be fixed after a breakage is confirmed. Is that a correct assumption?